app/policies/pair_request_policy.rb

```diff
@@ -0,0 +1,34 @@
+class PairRequestPolicy < ApplicationPolicy
+  alias_method :pair_request, :record
+
+  def index?
```

Comment:
left comment related to super already. I am thinking how (because our "user" is named differently per records) and we also have an invitee. What i'd like is to have an an `is_owner?` method to consolidate some of these.